Makefile

```diff
@@ -184,7 +184,7 @@ push-aws-dry: push
 
 push-gce-run: push
 	ssh ${TARGET} sudo cp /tmp/nodeup /home/kubernetes/bin/nodeup
-	ssh ${TARGET} sudo SKIP_PACKAGE_UPDATE=1 /home/kubernetes/bin/nodeup --conf=/var/cache/kubernetes-install/kube_env.yaml --v=8
+	ssh ${TARGET} sudo SKIP_PACKAGE_UPDATE=1 /home/kubernetes/bin/nodeup --conf=/var/lib/toolbox/kubernetes-install/kube_env.yaml --v=8
```

Comment:
This cause I figure container optimized OS will become the norm in GCE (correct me if I'm wrong)